modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/TSDBIndexingIT.java

```diff
@@ -456,16 +456,17 @@ public void testTrimId() throws Exception {
         {
             Instant time = Instant.now();
             for (int i = 0; i < numBulkRequests; i++) {
-                BulkRequest bulkRequest = new BulkRequest(dataStreamName);
-                for (int j = 0; j < numDocsPerBulk; j++) {
-                    var indexRequest = new IndexRequest(dataStreamName).opType(DocWriteRequest.OpType.CREATE);
-                    indexRequest.source(DOC.replace("$time", formatInstant(time)), XContentType.JSON);
-                    bulkRequest.add(indexRequest);
-                    time = time.plusMillis(1);
+                try (BulkRequest bulkRequest = new BulkRequest()) {
```

Comment:
`new BulkRequest(dataStreamName)` became `new BulkRequest()` -- does it matter?

Reply:
It probably doesn't, but I definitely didn't mean to remove it. Adding it back.